quotes/common.js

```diff
@@ -66,15 +92,15 @@ function createPopup(userkey, roles, userdata) {
     // The roles of the user
     const roleHeader = popupBody.appendChild(document.createElement("div"))
     roleHeader.className = "popup_body_header"
-    var userRoles = userdata?.roles
+    const userRoles = userdata?.roles;
 
     if (userRoles && userRoles.length > 0) {
         const rolesProperties = Object.getOwnPropertyNames(roles)
         userRoles.sort(function(a, b) {
             return rolesProperties.indexOf(a) - rolesProperties.indexOf(b);
         })
 
-        roleHeader.innerText = userRoles.length == 1 ? "Role" : "Roles"
+        roleHeader.innerText = userRoles.length === 1 ? "Role" : "Roles"
```

Comment:
Oh yeah, the danger of the abstract equality comparison operator. I forgot about that _quirk_ of JavaScript.

Reply:
Yes *fun quirk*, doesn't matter to much here but good style wise